docs/EVM/precompiles.md

```diff
@@ -4,6 +4,66 @@ sidebar_position: 1
 
 # Precompiles
 
-Precompiles are special addresses that provide an interface to interact with the network from inside EVM environment.
+A precompile just means a common functionality for smart contracts which has been compiled, so Ethereum nodes can run this efficiently. From a contract's perspective this is just a single command like an opcode.
+The Frontier EVM used in Astar provides several useful precompiled contracts. These contracts are implemented in the platform itself as a native implementations. The precompiled contracts from address `0x01` through` 0x08` are the same as those in Ethereum. Astar additionally implements precompiled contracts starting from `0x5001`, to support new Astar features.
+
+## Ethereum Native Precompiles
+| Precompile | Address |
```

Comment:
Perhaps this is fine, I'm just not a fan of duplicating information from the source code.

I'd suggest that you also link to a runtime/precompiles.rs file and explain that exact addresses can always be found there.

Reply:
I agree, but since these are hardcoded addresses there should not be any address change. I have added link to source code like you suggested